exporter/prometheusexporter/log.go

```diff
@@ -0,0 +1,20 @@
+package prometheusexporter
+
+import (
+	"fmt"
+	"go.uber.org/zap"
+)
+
+type promLogger struct {
+	realLog *zap.Logger
+}
+
+func NewLog(zapLog *zap.Logger) *promLogger {
```

Comment:
Does it have to be exported?

Reply:
If you don't export,Some exceptions may occur that cause indicators to be lost,But you don't have any way to check it

Reply:
Not sure I get what you mean... The only place using this new function is prometheus.go, which is part of the same package. You could certainly have it unexported (newLog) instead, from what I can see. What am I missing?